lua/telescope/_extensions/file_browser/finders.lua

```diff
@@ -159,68 +161,135 @@ end
 ---@field git_status boolean: show the git status of files (default: true)
 fb_finders.finder = function(opts)
   opts = opts or {}
+
+  deprecation_notices(opts)
+
+  local auto_depth_defaults = {
+    path_display = vim.F.if_nil(
+      require("telescope.config").pickers.find_files.path_display,
```

Comment:
I think this will error out with default telescope settings like `require('telescope').setup()`.

Reply:
Good catch :)